src/analysis/specific_limits.lean

```diff
@@ -695,6 +708,72 @@ end
 
 end normed_ring_geometric
 
+/-! ### Summability tests based on comparison with geometric series -/
+
+lemma summable_of_ratio_norm_eventually_le {α : Type*} [semi_normed_group α] [complete_space α]
+  {f : ℕ → α} {r : ℝ} (hr₀ : 0 ≤ r) (hr₁ : r < 1)
+  (h : ∀ᶠ n in at_top, ∥f (n+1)∥ ≤ r * ∥f n∥) : summable f :=
+begin
+  rw eventually_at_top at h,
+  rcases h with ⟨N, hN⟩,
+  rw ← @summable_nat_add_iff α _ _ _ _ N,
+  refine summable_of_norm_bounded (λ n, ∥f N∥ * r^n)
+    (summable.mul_left _ $ summable_geometric_of_lt_1 hr₀ hr₁) (λ n, _),
+  conv_rhs {rw [mul_comm, ← zero_add N]},
+  refine le_geom hr₀ n (λ i _, _),
+  convert hN (i + N) (N.le_add_left i) using 3,
+  ac_refl
+end
+
+lemma summable_of_ratio_test_tendsto_lt_one {α : Type*} [normed_group α] [complete_space α]
+  {f : ℕ → α} {l : ℝ} (hl₀ : 0 ≤ l) (hl₁ : l < 1) (hf : ∀ᶠ n in at_top, f n ≠ 0)
```

Comment:
Again, `hl₀` is not necessary (you can deduce it from `h`, as `l` is a limit of a nonnegative sequence)